activesupport/test/core_ext/duration_test.rb

```diff
@@ -111,7 +111,11 @@ def test_multiply
   def test_divide
     assert_equal 1.day, 7.days / 7
     assert_instance_of ActiveSupport::Duration, 7.days / 7
-    assert_equal 1, 1.day / 1.day
+    assert_equal 1.day, 1.day / 1.day
+
+    assert_equal 2.days, 4.days / 2.days
+    assert_equal 2.days, 4.days / 1 / 2.days
+    assert_instance_of ActiveSupport::Duration, 4.days / 1 / 2.days
```

Comment:
@fphilipe technically `7.days / 7` would return `86400` prior to Rails 5.1, however after #29971 it will return `1.day` with the appropriate correct parts so that if you add it to a time value and it crosses the DST boundary then it will work correctly whereas previously it would always add 86400 seconds.